content/docs/reference-react-component.md

```diff
@@ -143,500 +142,494 @@ If you need to interact with the browser, perform your work in `componentDidMoun
-
-```javascript
-getSnapshotBeforeUpdate(prevProps, prevState)
-```
-
-`getSnapshotBeforeUpdate()` is invoked right before the most recently rendered output is committed to e.g. the DOM. It enables your component to capture some information from the DOM (e.g. scroll position) before it is potentially changed. Any value returned by this lifecycle will be passed as a parameter to `componentDidUpdate()`.
-
-This use case is not common, but it may occur in UIs like a chat thread that need to handle scroll position in a special way.
-
-A snapshot value (or `null`) should be returned.
-
-For example:
-
-`embed:react-component-reference/get-snapshot-before-update.js`
-
-In the above examples, it is important to read the `scrollHeight` property in `getSnapshotBeforeUpdate` because there may be delays between "render" phase lifecycles (like `render`) and "commit" phase lifecycles (like `getSnapshotBeforeUpdate` and `componentDidUpdate`).
-
-* * *
-
-### Error boundaries
-
-[Error boundaries](/docs/error-boundaries.html) are React components that catch JavaScript errors anywhere in their child component tree, log those errors, and display a fallback UI instead of the component tree that crashed. Error boundaries catch errors during rendering, in lifecycle methods, and in constructors of the whole tree below them.
-
-A class component becomes an error boundary if it defines either (or both) of the lifecycle methods `static getDerivedStateFromError()` or `componentDidCatch()`. Updating state from these lifecycles lets you capture an unhandled JavaScript error in the below tree and display a fallback UI.
-
-Only use error boundaries for recovering from unexpected exceptions; **don't try to use them for control flow.**
-
-For more details, see [*Error Handling in React 16*](/blog/2017/07/26/error-handling-in-react-16.html).
-
-> Note
-> 
-> Error boundaries only catch errors in the components **below** them in the tree. An error boundary can’t catch an error within itself.
+Normalmente, los constructores de React sólo se utilizan para dos propósitos:
 
-### `static getDerivedStateFromError()`
-```javascript
-static getDerivedStateFromError(error)
-```
-
-This lifecycle is invoked after an error has been thrown by a descendant component.
-It receives the error that was thrown as a parameter and should return a value to update state.
-
-```js{7-10,13-16}
-class ErrorBoundary extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { hasError: false };
-  }
-
-  static getDerivedStateFromError(error) {
-    // Update state so the next render will show the fallback UI.
-    return { hasError: true };
-  }
-
-  render() {
-    if (this.state.hasError) {
-      // You can render any custom fallback UI
-      return <h1>Something went wrong.</h1>;
+* Para inicializar  un state local asignando un objeto al `this.state`.</li> 
+    
+    * Para enlazar [ gestores de eventos ](/docs/handling-events.html) a una instancia.</ul> 
```

Comment:
Spacing looks off here, can you please review what happened here?

Reply:
Not sure, why all this whitespaces i just downloaded Crowdin translate, i'll check it out tomorrow, thanks for reviewing mate